### DISCUSSION TITLE
HEIMAN HS2IRC

### DISCUSSION THREAD

**Discussion by Tony-Makarony** [Koenkk/zigbee2mqtt#7133] [Devices]
Good day. There are problems setting up this device. In zigbee2mqtt, the device does not respond to commands in the mqtt explorer. Can anyone tell me in more detail how to do this? I really look forward to help.

**Comment by coraxxx** [comment]
Same problem. Can somebody help?

**Comment by mslotema** [comment]
Hi,
Did you try setting the topic in the MQTT explorer:

`zigbee2mqtt/[name]/set`

The payload (json) is the actual command. Here is one for a device I'm working on:

`
{"brightness":100,"color_mode":"color_temp","color_temp":50,"state":"ON"}
`

Obviously this can (and probably will) be different from yours.

Cheers
